### PR TITLE
simplify handling messages

### PR DESCRIPTION
**Pull request by rkondratowicz**
Also added missing tests

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/slc/services/servicebus/MessageProcessorTest.java [hunk 1]
```diff
@@ -24,24 +22,23 @@
...
-        action = (msg) -> SUCCESS;
+        given(sendLetterService.send(any())).willReturn(SUCCESS);
         given(messageReceiver.receive()).willReturn(message);
 
         // when
-        processor.handle(action);
+        processor.process();
 
         // then
         verify(messageReceiver, times(1)).complete(any());
```
Comment (nitinprabhu) [inline src/test/java/uk/gov/hmcts/reform/slc/services/servicebus/MessageProcessorTest.java:44]: could you add `verifyNoMoreInteractions` for test cases

Comment (rkondratowicz) [reply src/test/java/uk/gov/hmcts/reform/slc/services/servicebus/MessageProcessorTest.java:44]: perhaps separately, not related to this refactoring

File: src/main/java/uk/gov/hmcts/reform/slc/services/SendLetterJob.java [hunk 2]
```diff
@@ -4,53 +4,22 @@
...
-                // TODO: encrypt & sign
-                ftpUploader.upload(pdf);
-
-                return SUCCESS;
-            } catch (Exception exc) {
-                logger.error(exc.getMessage(), exc.getCause());
-
-                return FAILURE;
-            }
-        });
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/slc/services/SendLetterJob.java:54]: this class is not doing much now

**Review by doncem — APPROVED** [review]
oh noz 🙀 , drastic restructure

**Pull request merged**